### DISCUSSION TITLE
Use categories in filters

### DISCUSSION THREAD

**Discussion by r4fx** [itx-informationssysteme/jobapplications#138] [Ideas]
It would very handy to have an option to use system categories for fields like `career_level` and `division` and then use these categories in filters. 

Generally speaking would connecting system categories with filters would improve this plugin massively with its flexibility.

**Comment by ghost** [comment]
Hi!
You may extend the Posting model as you wish and hide the existing fields, you can even use the ModifyGoogleForJobsDataEvent to map your custom categories to Google for Jobs values.

You can also use these for filters, see here: https://docs.typo3.org/p/itx/jobapplications/main/en-us/Developer/Index.html#custom-filters

A general option for this is not planned as we need the `career_level` field to be static and in sync with Google for Jobs field values.
